index.js

```diff
@@ -127,25 +145,23 @@ const drawCommand = (message) => {
   });
 };
 
-/*
-const inventoryCommand = () => {
-
-if user draws a card
-store name of card to firebase under the key cardName
-when the same user draws a second card, store the name of the second card under the previous draw within cardName
-continue to do this to all additional cards
-
-user{
-  userID{
-    lastDrawTime: xxxx
-    username: ""
-    cardName: "Blue Eyes White Dragon", "Red Eyes Black Dragon"
-  }
-}
-
+const inventoryCommand = (message) => {
+  var ref = db.ref("users");
+  ref.once("value", function (snapshot) {
+    const userData = snapshot.val();
+    const userIdentification = message.author.id;
+    const userInventoryAccess = userData[userIdentification].inventory;
+    let cardIDs = Object.keys(userInventoryAccess);
+    let inventoryList = "";
+
+    for (let i = 0; i < cardIDs.length; i++) {
```

Comment:
so `for`s have been handy throughout coding classes, but now they're a code smell.

A code smell is something that indicates something is wrong or can be simpler. Seeing a `let` is a code smell, for example. The smell is not always correct, since there are valid reasons for `for` and `let`, but it serves its purpose by making you think if you really need that.

Code smells usually have a typical solution. For example, `let`s should be `const`s, usually. With a `for` loop, you might be able to use one of the ES6 array loop functions.

Every smell has a reason for being a smell, and the reason for this smell is that `for` loops are very generic and very verbose -- most of the time you just want to loop over an array, and the ES6 array loop functions are custom made to do that one thing, unlike a `for` loop which can do a billion things.

Check out the "Converting a for loop to forEach" example for [.forEach in MDN docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) and see how you might apply that to `cardIDs`.